code/modules/surgery/organs/organ_internal.dm

```diff
@@ -239,6 +239,95 @@
 	S.reagents.add_reagent("salbutamol", 5)
 	return S
 
+/obj/item/organ/internal/tongue
+	name = "tongue"
+	desc = "A fleshy muscle mostly used for lying."
+	icon_state = "lungs"
+	zone = "head"
```

Comment:
Why `zone = "head"` when you can actually put them in mouth? It's supported by some of existing organs and organ manipulation for mouth is already a thing.